core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1808,6 +1811,31 @@ public ServerBuilder http1HeaderNaming(Http1HeaderNaming http1HeaderNaming) {
         return this;
     }
 
+    /**
+     * Sets the interval between reporting unhandled exceptions which is not handled
+     * by any decorators or services, such as {@link LoggingService}.
+     * @param unhandledExceptionsReportInterval the interval between reports, or
+     *                                        {@link Duration#ZERO} to disable this feature
+     * @throws IllegalArgumentException if specified {@code unhandledExceptionsReportInterval} is negative.
+     */
+    public ServerBuilder unhandledExceptionsReportInterval(Duration unhandledExceptionsReportInterval) {
```

Comment:
How about renaming the parameter to `interval` like the other one for brevity?

Reply:
Great idea !